functests/utils/consts.go

```diff
@@ -24,7 +30,25 @@ func init() {
 		PerformanceProfileName = "performance"
 	}
 
+<<<<<<< HEAD
 	NodesSelector = os.Getenv("NODES_SELECTOR")
+=======
+	if discovery.Enabled() {
+		performanceProfile, err := profiles.GetByNodeLabels(
+			map[string]string{
+				fmt.Sprintf("%s/%s", LabelRole, RoleWorkerCNF): "",
+			},
+		)
+		if err != nil {
+			klog.Error("Failed finding an existing performance profile in discovery mode", err)
```

Comment:
So is an error in logs, not connected to the tests error flow? Is this the standard way to expose tests results? I ask because I didn't see this approach in other places